rest/admin_api.go

```diff
@@ -358,13 +358,13 @@ func (h *handler) handleActiveTasks() error {
 func (h *handler) handleGetRawDoc() error {
 	h.assertAdminOnly()
 	docid := h.PathVar("docid")
-	includeDoc := h.getBoolQuery("include_doc")
-	redact := h.getOptBoolQuery("redact", true)
+	includeDoc := h.getOptBoolQuery("include_doc", true)
+	redact := h.getBoolQuery("redact")
```

Comment:
In the case where redact=true is set, and include_doc isn't explicitly set, we should return a result with redact=true/include_doc=false, instead of returning the error below.  The error should only be returned if users explicitly set redact=true and include_doc=true on the request.